ietf/templates/doc/document_html.html

```diff
@@ -34,7 +34,7 @@
               href="/feed/document-changes/{{ doc.name }}/">
         <meta name="description"
             {% if not snapshot and doc.get_state_slug == 'rfc' %}
-                content="{{ doc.title }} (RFC {{ doc.rfc_number }}{% if published %}, {{ published.time|date:'F Y' }}{% endif %}{% if obsoleted_by %}; obsoleted by {{ obsoleted_by|join:', ' }}{% endif %}"
+                content="{{ doc.title }} (RFC {{ doc.rfc_number }}{% if published %}, {{ published.time|date:'F Y' }}{% endif %}{% if obsoleted_by %}; obsoleted by {% for rel in obsoleted_by %}{{ rel.source.name|prettystdname}}{% if not forloop.last%}, {% endif %}{% endfor %}{% endif %})"
```

Comment:
This isn't quite right yet. For RFC8889, we get:
```
     <meta name="description"
            
                content="Multipoint Alternate-Marking Method for Passive and Hybrid Performance Monitoring (RFC 8889, August 2020; obsoleted by draft-ietf-ippm-rfc8889bis)"
            >
```
where instead of `draft-ietf-ippm-rfc8889bis` we should be getting `RFC 9342`